src/encoded/schemas/cart.json

```diff
@@ -31,6 +51,31 @@
             "type": "string",
             "pattern": "^(?!\\s)[\\S]*\\S$"
         },
+        "description": {
```

Comment:
we should add a dependency that makes Description a required property for cart in status "listed", "released", "revoked"